### PR TITLE
Added show amount of time added when advance to next round

### PR DESCRIPTION
**Pull request by ncnlinh**
WIP

### CODE REVIEW

File: src/sass/_game.scss [hunk 1]
```diff
@@ -68,6 +68,18 @@
   animation: blink normal 0.8s infinite ease-in-out; /* Opera */
 }
 
+.time-add {
+  color: #16A085;
+   -moz-transition: all 0.2s ease-in-out;
+  -webkit-transition: all 0.2s ease-in-out;
+  -o-transition: all 0.2s ease-in-out;
+  -ms-transition: all 0.2s ease-in-out;
+  transition: all 0.2as ease-in-out;
```
Comment (yangshun) [inline src/sass/_game.scss]: I think it'll be good to shift all the vendor prefixes stuff into a mixin.

Comment (yangshun) [inline src/sass/_game.scss]: Refer to https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss

Comment (ncnlinh) [inline src/sass/_game.scss]: :+1:
Good practice, will do.

**Comment by khankuan** [comment]
Closed due to change in game logic

**Pull request closed without merging**